packages/cutler/lib/commands/print_versions_command.dart

```diff
@@ -14,12 +14,30 @@ class PrintVersionsCommand extends Command<int> {
 
   @override
   int run() {
-    final shorebirdHash = argResults!.rest.first;
+    late final String shorebirdHash;
+    if (argResults!.rest.isEmpty) {
+      print('No Shorebird hash provided, using `origin/stable`.');
+      shorebirdHash = 'origin/stable';
+    } else {
+      shorebirdHash = argResults!.rest.first;
+    }
+
     final shorebirdFlutter = Repo.shorebird
         .contentsAtPath(shorebirdHash, 'bin/internal/flutter.version');
     final shorebird = getFlutterVersions(shorebirdFlutter);
     print('Shorebird $shorebirdHash:');
     printVersions(shorebird, 2);
+
+    final flutterForkpoint = Repo.flutter.getForkPoint(shorebird.flutter.hash);
+    // This is slightly error-prone in that we're assuming that our engine and
+    // buildroot forks started from the correct commit.  But I'm not sure how
+    // to determine the forkpoint otherwise.  engine and buildroot don't have
+    // a stable branch, yet they do seem to "branch" for stable releases at the
+    // x.x.0 release.
+    final forkpoints = getFlutterVersions(flutterForkpoint.hash);
+    print('Forkpoints:');
+    printVersions(forkpoints, 2);
```

Comment:
Potentially out-of-scope for this PR, but named arguments would make `printVersions` a bit more readable. I'm not totally sure what `forkpoints` is (I presume a list of the form `['1.0.0', '1.1.0', ...]` but am not sure) and don't know what the significance of `2` is.

Reply:
forkpoints is a VersionSet.  I'll make indent be a named arg for now.